core/common/src/main/java/alluxio/security/authorization/PermissionStatus.java

```diff
@@ -108,34 +104,39 @@ public static PermissionStatus getDirDefault() {
         .DEFAULT_FS_FULL_PERMISSION));
   }
 
-  // TODO(binfan): remove remote parameter by making two different get methods
   /**
    * Creates the {@link PermissionStatus} for a file or a directory.
    *
    * @param conf the runtime configuration of Alluxio
-   * @param remote true if the request is for creating permission from client side, the
-   *               username binding into inode will be gotten from {@code AuthenticatedClientUser
-   *               .get().getName()}.
-   *               If the remote is false, the username binding into inode will be gotten from
-   *               {@link alluxio.security.LoginUser}.
    * @return the {@link PermissionStatus} for a file or a directory
    * @throws java.io.IOException when getting login user fails
    */
-  public static PermissionStatus get(Configuration conf, boolean remote) throws IOException {
-    if (!SecurityUtils.isSecurityEnabled(conf)) {
+  public static PermissionStatus getDefault(Configuration conf) throws IOException {
+    if (!SecurityUtils.isAuthenticationEnabled(conf)) {
       // no authentication, every action is permitted
       return new PermissionStatus("", "", FileSystemPermission.getFullFsPermission());
     }
-    if (remote) {
-      // get the username through the authentication mechanism
-      User user = AuthenticatedClientUser.get(conf);
-      if (user == null) {
-        throw new IOException(ExceptionMessage.AUTHORIZED_CLIENT_USER_IS_NULL.getMessage());
-      }
-      return new PermissionStatus(user.getName(), CommonUtils.getPrimaryGroupName(conf,
-          user.getName()), FileSystemPermission.getDefault().applyUMask(conf));
+    // get the username through the authentication mechanism
+    User user = AuthenticatedClientUser.get(conf);
+    if (user == null) {
+      throw new IOException(ExceptionMessage.AUTHORIZED_CLIENT_USER_IS_NULL.getMessage());
     }
+    return new PermissionStatus(user.getName(), CommonUtils.getPrimaryGroupName(conf,
+        user.getName()), FileSystemPermission.getDefault().applyUMask(conf));
+  }
 
+  /**
+   * Creates the {@link PermissionStatus} for a file or a directory.
+   *
+   * @param conf the runtime configuration of Alluxio
+   * @return the {@link PermissionStatus} for a file or a directory
+   * @throws java.io.IOException when getting login user fails
+   */
+  public static PermissionStatus getAsLoginUser(Configuration conf) throws IOException {
```

Comment:
fromLoginUser?

so that the invocation looks like `PermissionStatus.fromLoginUser`